### DISCUSSION TITLE
GPU metrics

### DISCUSSION THREAD

**Discussion by thisiskazem** [sustainable-computing-io/kepler#1338] [General]
I've installed kepler on a bare-metal server using rpm. My server has a graphic card installed. How should i check if the enable-gpu=true or not because I cannot see the metrics related to gpu.

**Comment by rootfs** [comment]
can you post the kepler pod? `kubectl get -n kepler daemonset/kepler-exporter -o json`. The `args` field will show if the gpu option is turned on

**Comment by thisiskazem** [comment]
As I told before, I didn't implement it on kubernetes cluster but on a linux distribution using RPM.
